source/reference/ejson/serialize.txt

```diff
@@ -0,0 +1,171 @@
+.. _mongosh-ejson-serialize:
+
+=================
+EJSON.serialize()
+=================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/links-urls/ejson.rst
+
+The ``EJSON.serialize()`` method converts a BSON object to an Extended
+JSON string and then reparses the string as a JSON object.
+
+MongoDB stores data using BSON. Many external data transformation
+applications use JSON. You can use ``EJSON.serialize()`` to convert BSON
+to JSON and save the output for those external applications.
+
+Syntax
+------
+
+The method has this syntax:
+
+.. code-block:: javascript
+   :copyable: false
+
+   EJSON.serialize( object, [ options ] )
+
+Method Fields
+-------------
+
+The method takes the following fields:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Field
+     - Type
+     - Necessity
+     - Description
+
+   * - ``object``
+     -  BSON object
+     -  Required
+     -  BSON object to convert. For example, an array of documents.
+
+   * - ``options``
+     -  string
+     -  Optional
+     -  Modifies the output object :ref:`types <mongo-shell-data-type>`.
```

Comment:
Since the only option is boolean and it isn't on the types page, I don't think the link provides value here.

Reply:
This is boiler plate from the other pre-existing EJSON pages (which I didn't write). And, it is correct because the Boolean controls the output object type (JSON or BSON), as stated in the text.